src/styl/_components/_inlinemenu.styl

```
@@ -0,0 +1,51 @@
+// Inline Menu
+//
+// Set all elements of list in display flex, justify and align items center.
+//
+// This class can be used to make a inline menu.
+//
+// Markup: inlinemenu.twig
+//
+// Styleguide: Components.inlinemenu
+
+.menu
```

Ok so we want to have the filename same as the classname, so:

```suggestion
.inlinemenu
```